### PR TITLE
Curated training: remove easiest 30%, train on hard 70%

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
Easy samples (simple single-foil, low AoA) contribute little learning signal. Phase 1: Quick 5-min baseline to rank samples by loss. Phase 2: Train fresh model on only the hardest 70% of samples for 25 min. More gradient signal per sample.

## Instructions
Phase 1 (5 min): Train baseline ~12 epochs, evaluate per-sample training loss. Rank samples.
Phase 2 (25 min): Create filtered dataset with top-70% hardest samples. Train fresh model from scratch.

```python
# Phase 1: Quick ranking
model1 = create_model()
train(model1, wall_clock_limit=300)  # 5 min
# Compute per-sample loss on full training set
sample_losses = evaluate_per_sample(model1)
threshold = np.percentile(sample_losses, 30)  # bottom 30% = easy
hard_indices = [i for i, l in enumerate(sample_losses) if l >= threshold]

# Phase 2: Train on curated set
curated_dataset = Subset(train_dataset, hard_indices)
model2 = create_model()
train(model2, curated_dataset, wall_clock_limit=1500)  # 25 min
```

Run: `python train.py --agent senku --wandb_name "senku/curated-hard-70" --wandb_group curated-data`

## Baseline (true mean): val_loss ≈ 2.243 ± 0.025
---
## Results
_To be filled by student_

### CODE REVIEW

**Comment by tcapelle** [comment]
Closing — restarting with updated entrypoints.
